packages/api/src/router/developer.ts

```diff
@@ -37,4 +37,48 @@ export const developerRouter = createTRPCRouter({
       data: { upvotes: { increment: 1 } },
     });
   }),
+
+  sarinali_info: publicProcedure.query(async ({ ctx }) => {
+    let upvotes = 0;
+    // check if exist in table
```

Comment:
nit: I generally don't like comments in code, prefer good variable naming and simplifying code. The code here probably doesn't need comments since you have good variable names